### PR TITLE
Testing brew install

### PR DESCRIPTION
**Pull request by gabriel**

### CODE REVIEW

File: go/client/cmd_launchd_osx.go [hunk 1]
```diff
@@ -280,3 +281,44 @@ func (v *CmdLaunchdStatus) Run() error {
...
+		return nil
+	}
+
+	binPath, err := filepath.Abs(os.Args[0])
+	if err != nil {
+		return err
+	}
+	plistArgs := []string{"service"}
+
+	envVars := make(map[string]string)
```
Comment (maxtaco) [inline go/client/cmd_launchd_osx.go]: this code is dupe'd from above.  can it be factored into one function?

File: go/keybase/main.go [hunk 2]
```diff
@@ -107,6 +107,14 @@ func mainInner(g *libkb.GlobalContext) error {
 				return err
 			}
 		}
+
+		if libkb.IsBrewBuild {
```
Comment (maxtaco) [inline go/keybase/main.go]: put in else so that auto-fork take precendence

File: go/keybase/main.go [hunk 3]
```diff
@@ -107,6 +107,14 @@ func mainInner(g *libkb.GlobalContext) error {
...
+			err := client.BrewAutoInstall(g)
+			if err != nil {
+				g.Log.Errorf("Error trying to install service: %s", err)
```
Comment (maxtaco) [inline go/keybase/main.go]: do we want to error out here?

File: go/client/cmd_launchd_osx.go (hunk 1 above)
Comment (gabriel) [inline go/client/cmd_launchd_osx.go]: yeah

File: go/keybase/main.go (hunk 2 above)
Comment (gabriel) [inline go/keybase/main.go]: yeah

File: go/keybase/main.go (hunk 3 above)
Comment (gabriel) [inline go/keybase/main.go]: yeah

**Pull request merged**